src/common/github/pull-request-test-workflow.ts

```diff
@@ -95,21 +95,21 @@ export class PullRequestTest extends Component {
    */
   static addToProject(project: javascript.NodeProject, options: PullRequestTestOptions & WithDefaultWorkflow) {
     const hasDefaultGithubWorkflows = options.hasDefaultGithubWorkflows ?? true
-    const lighthouse = options.lighthouse ?? true
+    const isLighthouseEnabled = options.isLighthouseEnabled ?? true
     const {runsOn, outdir, workflowNodeVersion} = options
 
     if (!hasDefaultGithubWorkflows) {
       return
     }
 
     if (project.github) {
-      new PullRequestTest(project.github, {lighthouse, runsOn, workflowNodeVersion})
+      new PullRequestTest(project.github, {runsOn, workflowNodeVersion})
```

Comment:
Is it correct, that we want to have the lighthouse workflow only if the project is a subproject in a monorepo?
Why remove it here - for standalone project?

Reply:
I would say that rarely we would want to deploy the Playwright as a separate project. Let's keep this possibility if that's not too much work.

Reply:
I did it by mistake, thanks

Reply:
Ok, understood.